crates/bevy_reflect/bevy_reflect_derive/src/container_attributes.rs

```diff
@@ -151,11 +160,99 @@ impl ReflectTraits {
                         }
                     }
                 }
+                // Handles `#[reflect( alias = "Foo" )]`
+                NestedMeta::Meta(Meta::NameValue(pair)) => {
+                    let ident = pair
+                        .path
+                        .get_ident()
+                        .ok_or_else(|| syn::Error::new(pair.span(), "not a valid path"))?;
+
+                    // Closure that handles defining an alias on a generic type
+                    let try_handle_generic_alias = || -> syn::Result<()> {
+                        if is_generic {
+                            Err(syn::Error::new(ident.span(), "cannot specify non-generic aliases on generic types. Consider using `TypeRegistry::register_alias` instead"))
```

Comment:
```suggestion
                            Err(syn::Error::new(ident.span(), "aliases attribute cannot be used on generic types. Consider using `TypeRegistry::register_alias` instead"))
```

Reply:
Hm, maybe `alias attributes cannot be used on generic types. Consider using `TypeRegistry::register_alias` instead`?

Reply:
Yeah! One day I shall stop falling into English plurals traps (hopefully) 🤕

Reply:
All good! :D